astropy/units/quantity.py

```diff
@@ -192,6 +192,18 @@ def new_like(self, cols, length, metadata_conflicts='warn', name=None):
 
         return out
 
+    def get_sortable_cols(self):
```

Comment:
Is this needed, given that you define it below on the base info class?

Reply:
Good catch, this is a leftover from the original where the base method raised `NotImplementedError` and forced all classes to implement something (i.e. explicit opt-in).  In the end I decided that just returning the parent column is an OK default and classes that don't work have to implement the right thing or else opt-out with NotImplementedError.